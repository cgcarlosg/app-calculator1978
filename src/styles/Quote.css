.quote-body {
  padding: 0 90px;
  display: flex;
  align-items: center;
  height: 85%;
  color: black;
}

.quote {
  font-style: italic;
}

.author {
  text-align: right;
  padding: 10px 60px;
}
