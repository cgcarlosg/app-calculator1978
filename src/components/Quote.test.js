import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('Renders the Quote page', () => {
  const quote = renderer
    .create(<Quote />)
    .toJSON();
  expect(quote).toMatchSnapshot();
});
