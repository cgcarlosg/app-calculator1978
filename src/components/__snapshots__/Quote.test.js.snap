// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Quote page 1`] = `
<div
  className="quote-body"
>
  <h1>
    <q
      className="quote"
    >
      There are many men of principle in both parties in America, but there is no party of principle.
    </q>
    <div
      className="author"
    >
      âŽ¯ Alexis de Tocqueville
    </div>
  </h1>
</div>
`;
